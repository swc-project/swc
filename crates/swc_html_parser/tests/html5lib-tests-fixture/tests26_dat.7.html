<!DOCTYPE html><body><b><nobr>1<ins><nobr></b><i>2