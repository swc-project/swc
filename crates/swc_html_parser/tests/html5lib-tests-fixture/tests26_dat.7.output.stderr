(1,42): unexpected-start-tag-implies-end-tag
(1,42): adoption-agency-1.3
(1,46): adoption-agency-1.3
(1,50): expected-closing-tag-but-got-eof